packages/core/.babelrc.js

```diff
@@ -0,0 +1,13 @@
+const modules = process.env.NODE_ENV === 'test' ? 'commonjs' : false
+
+module.exports = {
+  presets: [
+    ['env', {
+      modules,
+      loose: true,
```

Comment:
`forceAllTransforms: true`

Reply:
was thinking about it, but aint sure if there is any benefit in that if we do not provide non-default targets

Reply:
I think it's same target like `uplify` before. At least if defaults will be changed then nothing will be broken.

Reply:
> I think it's same target like uplify before. 

Just a confirmation - that's exactly what it is

> At least if defaults will be changed then nothing will be broken.

Good point, gonna add this later.

Reply:
I meant semantically the same too.

Reply:
Am I understanding [`forceAllTransforms`](https://github.com/babel/babel-preset-env/blob/master/README.md#forcealltransforms) correctly in that it essentially causes `babel-preset-env` to behave like `babel-preset-es2015`?

Reply:
Kinda, its job is to output pure es5 which can be ofc understood by subsequent tools (like uglifyjs)

Preset2015 has only es2015 plugins in it, but preset-env has also other yearly presets and also it has (or soon it will, aint sure) shipped proposals plugins (like when browsers implement language features from stage 3 or something).